tests/collections/asr/test_asr_interctc_models.py

```diff
@@ -234,7 +234,10 @@ def __getitem__(self, idx):
                 if model_class is EncDecCTCModel:
                     assert output[0].shape == logprobs.shape
 
-            trainer = pl.Trainer(max_epochs=1)
+            ## Explicitly pass acclerator as cpu, since deafult val in PTL >= 2.0 is auto and it picks cuda
```

Comment:
@Kipok Could you take a look at this test and the bugs (?) that may be there under these cases.

Reply:
I'm a bit out of context - what exactly is the error that's happening here? Using CPU should be fine I guess, since we do not test anything GPU-specific here. Or does it not work in that way?